Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -909,6 +922,17 @@ open class ComposerVC: _ViewController,
         guard let maxMessageLength = channelConfig?.maxMessageLength else { return true }
         return textView.text.count + (text.count - range.length) <= maxMessageLength
     }
+    
+    public func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        guard let text = textView.text,
+              let range = Range(characterRange, in: text)
+        else { return true }
+        
+        let string = String(text[range].replacingOccurrences(of: "@", with: ""))
+        let user = content.mentionedUsers.first(where: { $0.name == string })
+        didTapOnMentionedUser(user)
+        return true
+    }
```

Comment:
This is duplicated logic in the Message Content View. We need to reuse this logic somehow. Maybe you can extract this to a component like `TextViewMentionedUserDetector` 🤔